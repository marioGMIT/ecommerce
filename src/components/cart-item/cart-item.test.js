import { shallow, mount, render } from 'enzyme';
import CartItem from './cart-item.component';

import React from 'react';

const mockItem = {
    imageUrl: 'www.testImage.com',
    price: 10,
    name: 'hats',
    quantity: 2
  };

it('Rendering CartItem component', () => {
    expect(shallow(<CartItem item={mockItem} />)).toMatchSnapshot()
})