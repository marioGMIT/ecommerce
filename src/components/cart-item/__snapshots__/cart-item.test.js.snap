// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rendering CartItem component 1`] = `ShallowWrapper {}`;
